UT4MasterServer/Program.cs

```diff
@@ -153,6 +153,11 @@ public static void Main(string[] args)
 
 		var app = builder.Build();
 
+		IConfiguration configuration = app.Configuration;
+		IWebHostEnvironment environment = app.Environment;

```

Comment:
There is no need to have these two variables since there's only one use of them.

Reply:
Done